arangodb-net-standard/GraphApi/PutGraphEdgeResult.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace ArangoDBNetStandard.GraphApi
+{
+    public class PutGraphEdgeResult
```

Comment:
Is it normal there is no `_from` and `_to`?

Reply:
So this is normal, the endpoint does not return `_from` and `_to`. However it returns `_oldRev`. I have tested the endpoint and here is an example response:

```json
{
    "error": false,
    "code": 202,
    "edge": {
        "_id": "myedgeclx/4424498",
        "_key": "4424498",
        "_oldRev": "_ZpaD8Hm--_",
        "_rev": "_ZpaGocW--_"
    }
}
```

I will open a new issue.